src/store/deployments/actions.js

```diff
@@ -386,7 +386,7 @@ export const prepareDeployments = (experiments, projectId, routerProps) => (
     .then(() => {
       dispatch(prepareDeploymentsDataLoaded());
 
-      routerProps.history.push(`/projetos/${projectId}`);
+      routerProps.history.push(`/projetos/${projectId}/pre-implantacao`);
```

Comment:
@LuanEdCosta bem, aqui parece que os deployments recém-criados não são adicionado ao state (provavelmente por conta do `history.push`, que vai disparar fetchDeployments).
Devo adicioná-los ao state, ou não faz diferença?

Reply:
Acho que não faz diferença, quando abrir a tela de pré-implantação os deployments vão ser buscados e o state vai ser atualizado. Só tem um detalhe: quando a tela de pré-implantação for aberta o primeiro deployment vai ser selecionado porque não foi passado nenhum ID na URL